sycl/source/detail/scheduler/scheduler.hpp

```diff
@@ -715,6 +715,38 @@ class Scheduler {
 
   friend class Command;
   friend class DispatchHostTask;
+
+  class StreamBuffers {
```

Comment:
It's all public. Why is it class then? Suggest switching to struct here.

Reply:
Agree, switched to structure